src/test/java/tests/UtilityTest.java

```diff
@@ -55,13 +55,26 @@ public void stripQuotesTest() {
     }
 
     @Test
-    public void dateManipulationTest() {
+    public void parseHTTPLastModifiedDateTest() {
         Calendar testDate = Calendar.getInstance();
         testDate.set(1994, Calendar.NOVEMBER, 15, 12, 45, 26);
         testDate.setTimeZone(TimeZone.getTimeZone("GMT"));
 
         assertEquals(testDate.getTime().toString(),
-                parseHTTPLastModifiedDate("Tue, 15 Nov 1994 12:45:26 GMT").toString());
+            parseHTTPLastModifiedDate("Tue, 15 Nov 1994 12:45:26 GMT").toString());
+
+        try {
+            assertEquals(testDate.getTime().toString(),
+                parseHTTPLastModifiedDate("some non-date string").toString());
```

Comment:
The input date is always given in English -- the problem was SimpleDateFormat trying to interpret the English input as Chinese, due to the default locale. :stuck_out_tongue: I think any input which causes a parse failure should work.